### ISSUE TITLE
Stephanie - July

### ISSUE THREAD

**Issue by bpbond** [PNNL-PREMIS/PREMIS-ghg#41]
(Getting a start on this now as things occur to me.)

- [ ] Flux measurements every 7-10 days; skip 'extra' controls every other time
- [ ] Download sapflow data ~ every 2 weeks, send to Nate and Charlotte
- [ ] Work with Ben, Roy, Pat on power cable for permanent chamber test, and do test
- [ ] Take a look at litter trap designs, finalize with me, order materials
- [ ] Probably dendrometer band installation (TBA)

**Comment by stephpenn1** [comment]
Also,

- [ ] Basic analysis of tree distance & Licor data
- [ ] Draft AGU abstract

**Comment by stephpenn1** [comment]
@bpbond would you like weather station data to be uploaded as CSV? They download as .dtf files originally.

**Comment by bpbond** [comment]
What are .dtf files? Are they text, i.e. is this just Onset's custom file extension for what is fundamentally a flat file? Or are they binary, and you need to export to csv?

**Comment by stephpenn1** [comment]
To the best of my knowledge, they seem binary. i.e. opening them as a Word file produces 65 pages of symbols. I can't easily rename as a CSV, but I will try to save it through the HOBOware as a CSV?

**Comment by bpbond** [comment]
OK. Hmm. How big is each binary file? Ideally we'd keep the raw binary but also a CSV version for easy importing...but maybe just the binary if we can parse it correctly. Thanks.

**Comment by stephpenn1** [comment]
The file from 6/21 (~10 days of data) is 15KB. I can export the files in HOBOware as CSV, so I can have a raw binary and CSV version.

**Comment by bpbond** [comment]
Hi @stephpenn1 as I start to move brain into work mode again, slowly–is the above list (and its checks) current from your point of view? Thanks.

**Comment by stephpenn1** [comment]
@bpbond yes, all unchecked tasks are started, though (especially on code, have a few questions on those) .. just did not feel confident checking them off yet.

**Comment by bpbond** [comment]
Thanks! Just trying to make sure I know where things stand.
